pkg/cmd/server/api/latest/helpers.go

```diff
@@ -140,21 +141,38 @@ func captureSurroundingJSONForError(prefix string, data []byte, err error) error
 
 // IsAdmissionPluginActivated returns true if the admission plugin is activated using configapi.DefaultAdmissionConfig
 // otherwise it returns a default value
-func IsAdmissionPluginActivated(reader io.Reader, defaultValue bool) (bool, error) {
+// It also returns true if configapi.DefaultAdmissionConfig is passed or false if the admission plugin's
+// configuration is passed to avoid passing configapi.DefaultAdmissionConfig to admission plugins.
+func IsAdmissionPluginActivated(reader io.Reader, defaultValue bool) (bool, bool, error) {
```

Comment:
What about making this an enumerated string?  

 1. EnabledWithDefaultAdmissionConfig
 2. EnabledWithCustomConfig
 2. Disabled

right now you have four possible cases and one of them isn't useful.